pennylane/ops/op_math/decompositions/unitary_decompositions.py

```diff
@@ -125,8 +126,10 @@ def two_qubit_decomposition(U, wires):
     where :math:`A, B, C, D` are :math:`SU(2)` operations, and the rotation angles are
     computed based on features of the input unitary :math:`U`.
 
-    For the 2-CNOT case, the decomposition is currently not supported and will
-    instead produce a 3-CNOT circuit like above.
+    For the 2-CNOT case, the decomposition is based on the
+    real-trace criterion of Shende-Bullock-Markov (https://arxiv.org/pdf/quant-ph/0308045).
```

Comment:
This reference is already cited above, so I would replace with

```suggestion
    real-trace criterion of Proposition III.3 in reference (2).
```